src/infra/docs/docs-rs/README.md

```diff
@@ -0,0 +1,14 @@
+# docs.rs
+
```

Comment:
This should have a sub-heading that is more descriptive than just the list of links.
```suggestion
### External Links
```